analysis_missing_concepts.py

```diff
@@ -0,0 +1,53 @@
+"""Analysis of issues to do with concepts missing in the latest OMOP vocab tables"""
+from typing import Dict, List, Set
+
+from backend.db.utils import get_db_connection, sql_in, sql_query
+
+
+def check_known_problematic_csets(problem_csets: List[int] = [1000011135, 1000020788, 1000027123, 1000039519,
```

Comment:
Checked to see if any of the csets taking a long time / not expanding had any deleted concepts. None were found.